.flake8.jinja

```diff
@@ -1 +1,10 @@
-{%- include "vendor/oca-addons-repo-template/.flake8" -%}
+[flake8]
```

Comment:
I see that you have hardcoded this. The idea is to use the same linter configuration here that on oca-addons-repo-template, so instead you have to use the new `
.pre-commit-config.yaml` from vendor submodule and remove these ones.

Reply:
if you replace for ruff, probably this file will not be needed anymore.

Reply:
Yes, that's the idea. In fact, the `.pre-commit-config.yaml` from oca-addons-repo-template has changed to ruff (not the one being templated):

https://github.com/OCA/oca-addons-repo-template/blob/8ad97a63fde3efe033fb262a6fb73d08a897e73d/.pre-commit-config.yaml#L24